libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtils.kt

```diff
@@ -1569,10 +1570,41 @@ internal object MapboxRouteLineUtils {
     internal fun getAlternativeRouteDeviationOffsets(
         metadata: AlternativeRouteMetadata
     ): Double {
-        return distinctGranularDistancesProvider(metadata.navigationRoute)?.let {
+        return distinctGranularDistancesProvider(metadata.navigationRoute)?.let { distances ->
```

Comment:
Shouldn't unit tests be adjusted/added?

Reply:
Definitely, I run out of time yesterday :+1: